### PR TITLE
TOK-383: in-progress-submit-kyc

### PR DESCRIPTION
**Pull request by franciscotobar**
[TOK-383](https://rsklabs.atlassian.net/browse/TOK-383)

### CODE REVIEW

File: src/app/collective-rewards/user/components/Button/BecomeABuilderButton.tsx [hunk 1]
```diff
@@ -24,34 +26,63 @@ const BuilderRegistrationButton = () => {
...
+    paused: PausedComponent,
+    deactivated: DeactivatedComponent,
+  }[builderState]
 }
 
-const getBuilderState = (builderStateFlags?: BuilderStateFlags): BuilderState => {
-  if (!builderStateFlags) return 'inProgress'
-  const { activated, communityApproved } = builderStateFlags
-  return activated && communityApproved ? 'active' : 'inProgress'
+const getBuilderState = (builder: Builder): ExtendedBuilderState => {
```
Comment (franciscotobar) [inline src/app/collective-rewards/user/components/Button/BecomeABuilderButton.tsx:74]: @antomor can you review this part :)

File: src/components/Modal/BecomeABuilderModal.tsx [hunk 2]
```diff
@@ -198,6 +198,9 @@ const Steps: FC = () => {
         </Typography>
         .
       </li>
+      <li className="mt-4 list-none font-normal">
+        *You need to use a different address in case you stopped being a builder.
```
Comment (antomor) [inline src/components/Modal/BecomeABuilderModal.tsx:202]: @franciscotobar Where did you get this text from?

File: src/app/collective-rewards/user/components/Button/BecomeABuilderButton.tsx (hunk 1 above)
Comment (antomor) [reply src/app/collective-rewards/user/components/Button/BecomeABuilderButton.tsx:73]: @franciscotobar LGTM 👍🏼

**Review by antomor — APPROVED** [review]
LGTM

File: src/components/Modal/BecomeABuilderModal.tsx (hunk 2 above)
Comment (franciscotobar) [reply src/components/Modal/BecomeABuilderModal.tsx:202]: Roger gave it to me:

image

**Pull request merged**